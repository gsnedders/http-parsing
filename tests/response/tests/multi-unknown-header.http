HTTP/1.1 200 OK
Connection: Close
X-Foobar: One
X-Foobar: Two
X-Foobar: Three

Test