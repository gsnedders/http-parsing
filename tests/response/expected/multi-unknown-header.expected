{
	"HTTP": {
		"version": "1.1",
		"status": 200,
		"reason": "OK"
	},
	
	"headers": {
		"connection": "close",
		"x-foobar": "One, Two, Three"
	},
	
	"body": "Test"
}